Makefile

```diff
@@ -2,7 +2,7 @@ RUNC_IMAGE=runc_dev
 RUNC_TEST_IMAGE=runc_test
 PROJECT=github.com/opencontainers/runc
 TEST_DOCKERFILE=script/test_Dockerfile
-BUILDTAGS=seccomp
+BUILDTAGS=seccomp selinux
```

Comment:
Shouldn't it just say SELinux is disabled.  I accidentally pushed it.  When I turned it on, I found it did not test my code anyways since the tests were happening inside of a container, where the processes thought SELinux was disabled.